src/Template/Template.php

```diff
@@ -189,7 +189,13 @@ public function start($name)
             );
         }
 
-        $this->sections[$name] = '';
+        if (isset($this->sections[$name])) {
+            $content = $this->sections[$name];
+            unset($this->sections[$name]);
+            $this->sections[$name] = $content;
```

Comment:
So what exactly is the purpose of this `if` block. It looks as though you are unsetting then setting the same data?